packages/engine-client/src/EngineBaseClient.ts

```diff
@@ -91,20 +91,22 @@ export class EngineBaseClient {
     params: EngineServerQueryRequestByType[TRequestType],
   ): Promise<EngineServerQueryResponseByType[TRequestType]> {
     const request = this.getQueryRequest(requestType, params);
-    const queryParams: Record<string, string | number> = {};
+    const requestBody: Record<string, any> = {};
     Object.keys(request).forEach((key) => {
       const value = request[key as keyof typeof request];
       // Remove null values and stringify
       if (value != null) {
-        queryParams[key] = String(value);
+        if (typeof value == 'boolean') {
+          requestBody[key] = String(value);
```

Comment:
I think backend currently expects strings for booleans -- would require some changes in some of the endpoints, think might be a bit out of scope, but lmk if you feel strongly and i can see what we can do in this pass. fwiw, we have plans for deprecating support for GET on queries at which point we can make this schema changes.

Reply:
actually -- the actual issue here is that the following queries expect strings for booleans (not that the backend can't handle json boolean):

```
MaxOrderSize.spot_leverage

MaxWithdrawable.spot_leverage

MaxLpMintable.spot_leverage
```
think the reason is so that it works using GET via query params. for now, we can just update the sdk to serialize those correctly and we wouldn't need this check anymore. will fix.

Reply:
we can remove it here (see last commit); we still need it in backend to support `GET`